google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/ITInstanceAdminTest.java

```diff
@@ -69,9 +69,7 @@ public void listInstances() {
     Instance instance =
         Iterators.getOnlyElement(
             instanceClient
-                .listInstances(
-                    Options.filter(
-                        "name:instances/" + env.getTestHelper().getInstanceId().getInstance()))
+                .listInstances(Options.filter("name=" + env.getTestHelper().getInstanceId()))
```

Comment:
I think this should be `"name:" + env.getTestHelper().getInstanceId()`

See https://cloud.google.com/spanner/docs/reference/rpc/google.spanner.admin.instance.v1#google.spanner.admin.instance.v1.ListInstancesRequest

Reply:
The problem here is that `:` operator is a contains and we have 3 instances that start with `spanner-testing`. That is why the test is failing.

By using the `=` operator we do an exact match instead, having a single instance returned. More on operators here: https://cloud.google.com/sdk/gcloud/reference/topic/filters